accelerator/core/apis.c

```diff
@@ -394,6 +394,33 @@ status_t api_fetch_txn_with_uuid(const ta_cache_t* const cache, const char* cons
   return ret;
 }
 
+status_t api_register_mam_channel(const ta_cache_t* const cache, const char* const obj, char** json_result) {
+  status_t ret = SC_OK;
+  ta_register_mam_channel_req_t* req = ta_register_mam_channel_req_new();
+  char uuid[UUID_STR_LEN];
+
+  ret = register_mam_channel_req_deserialize(obj, req);
+  if (ret) {
+    ta_log_error("Failed to deserialize request of 'api_register_mam_channel'\n");
```

Comment:
You don't have to mention the function name. Instead, the logger can generate the messages with the caller function.